charts/trino/values.yaml

```diff
@@ -439,3 +439,29 @@ ingress:
   #  - secretName: chart-example-tls
   #    hosts:
   #      - chart-example.local
+
+
+# Requires the prometheus-operator to be installed in the cluster
+# Requires jmx_exporter to be installed in the cluster
```

Comment:
I wonder how we should test this. Should there be a separate job in https://github.com/trinodb/charts/blob/main/.github/workflows/ci-cd.yaml, that would install these requirements? If we won't be testing this, I don't think it should be included in the chart.